website/source/docs/platform/k8s/helm.html.md

```diff
@@ -129,6 +129,15 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-server-extraconfig" href="#v-server-extraconfig">`extraConfig`</a> (`string: "{}"`) -A raw string of extra JSON or HCL configuration for Consul servers. This will be saved as-is into a ConfigMap that is read by the Consul server agents. This can be used to add additional configuration that isn't directly exposed by the chart.
```

Comment:
```suggestion
  * <a name="v-server-extraconfig" href="#v-server-extraconfig">`extraConfig`</a> (`string: "{}"`) -A raw string of extra JSON or HCL [configuration](/docs/agent/options.html) for Consul servers. This will be saved as-is into a ConfigMap that is read by the Consul server agents. This can be used to add additional configuration that isn't directly exposed by the chart.
```

Reply:
I've added the link and removed the HCL reference.